src/write_resume_data.cpp

```diff
@@ -126,7 +126,7 @@ namespace libtorrent {
 		if (!atp.trackers.empty())
 		{
 			entry::list_type& tr_list = ret["trackers"].list();
-			tr_list.push_back(entry::list_type());
+			tr_list.emplace_back(entry::list_type());
```

Comment:
yes, but below, it needs to be `entry::list_type`, can't be a generic default one (unless the default is a list, or...is it?)